Makefile

```diff
@@ -76,7 +76,6 @@ TAG ?= latest
 
 # options for generating crds with controller-gen
 CONTROLLER_GEN="${GOBIN}/controller-gen"
-CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
```

Comment:
can u explain this one pls?

Reply:
The flags did not exist anymore in controller-gen 0.10. I actually searched for when they were removed but did not find it. Is unfortunate that GitHub code search searches the default branch only and there it is not in anymore. I then just ran it and compared the output (= the JSON files created in deploy/crds) and did not see significant differences. Based on that I concluded that removing them is fine.

Reply:
alright, that's fair.